geos/tests/test.bats

```diff
@@ -0,0 +1,6 @@
+source "${BATS_TEST_DIRNAME}/../plan.sh"
+
+@test "Version matches" {
+  result="$(geos-config --version)"
```

Comment:
this works within this context.  however, when someone attempts `hab pkg exec core/geos ceos-config --version`, there's an error stating no `cat`.  We should add `core/coreutils` as a `pkg_deps`

Reply:
Good catch :)
Done now.